src/containers/Petition.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { fetchPetition } from 'actions/PetitionActions';
+import Petition from 'components/Petition';
+
+const PetitionContainer = React.createClass({
+  // When the component gets added to the DOM,
+  // fetch Petition if `id` is not defined.
+  componentDidMount () {
+    if (!this.props.id) {
+      this.props.fetchPetition(this.props.params.id);
+    }
+  },
+
+  render () {
+    return (
+      <div>
+        <Helmet title={this.props.title} />
```

Comment:
Defines the page `<title>` added to the title template for this page type, i.e. the petition title